packages/vc-verification/src/Ipfs-credential-resolver.ts

```diff
@@ -178,8 +182,11 @@ export class IpfsCredentialResolver implements CredentialResolver {
       await Promise.all(
         services.map(async ({ serviceEndpoint }) => {
           const credential = await this._ipfsStore.get(serviceEndpoint);
-          const vc = JSON.parse(credential);
-          delete vc.iat;
+          let vc;
+          if (!(credential.split('.').length === 3)) {
```

Comment:
Does this handles non-jwt credential? How jwt credential is handled?

Reply:
Yes, so to get a Verifiable Credential the string coming from IPFS should have VC. The JWT credential is handled if you fetch the OffChainClaim.